example/speedtest/pub/main.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) TrueBrain
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <TrueMQTT.h>
+#include <chrono>
+#include <iostream>
+#include <thread>
+#include <unistd.h>
+
+enum TIME_RES
+{
+    T_MICRO,
+    T_MILLI,
+    T_SECONDS
+};
+
+std::uint64_t getEpochUSecs()
+{
+    auto tsUSecs = std::chrono::time_point_cast<std::chrono::microseconds>(std::chrono::system_clock::now());
+    return static_cast<std::uint64_t>(tsUSecs.time_since_epoch().count());
+}
+
+int main(int argc, char* argv[])
+{
+    // Create a connection to a local broker over IPv4 and setup the MQTT-Stack
+    // =========================================================================
```

Comment:
If you need these kind of comments, please put the content in a function. As that is what you are actually doing with these comments :)

Reply:
Good idea..